### DISCUSSION TITLE
Prevent creating a protected branch from a non-protected branch

### DISCUSSION THREAD

**Discussion by tamlyn** [community/community#38707] [Repositories]
We have branch protection rules to protect `main` and anything matching `release/*`. We want to ensure that all code on a protected branch has been reviewed. That's fine for `main` but anyone can create `release/*` branch from any commit meaning the code on that branch may not have been reviewed. 

Is there any way to enforce that our `release/*` branches can only be created on the remote from a commit which is on `main` (or any protected branch)?

**Comment by ossanna16** [comment]
Hi there @tamlyn and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
